importer/importer.go

```diff
@@ -3,39 +3,14 @@
 package importer
 
 import (
-	"fmt"
-	"os"
-
 	chunker "gx/ipfs/QmWo8jYc19ppG7YoTsrr2kEtLRbARTJho5oNXFTR6B7Peq/go-ipfs-chunker"
-	"gx/ipfs/QmceUdzxkimdYsgtX733uNgzf1DLHyBKN6ehGSp85ayppM/go-ipfs-cmdkit/files"
 	ipld "gx/ipfs/Qme5bWv7wtjUNGsK2BNGVUFPKiuxWrsqrtvYwCLRw8YFES/go-ipld-format"
 
 	bal "github.com/ipfs/go-ipfs/importer/balanced"
 	h "github.com/ipfs/go-ipfs/importer/helpers"
 	trickle "github.com/ipfs/go-ipfs/importer/trickle"
 )
 
-// BuildDagFromFile builds a DAG from the given file, writing created blocks to
-// disk as they are created.
-func BuildDagFromFile(fpath string, ds ipld.DAGService) (ipld.Node, error) {
```

Comment:
While not currently used, I wouldn't necessarily delete this. It's not that much code and may be useful.

On the other hand... stating and then opening is racy.